live-build/misc/live-build-hooks/82-upgrade-repository.binary

```diff
@@ -92,9 +92,32 @@ cp config/hooks/template.ctl delphix-entire.ctl
 DEPENDS=$(chroot binary \
 	dpkg-query -Wf '${Package} (=${Version}), ' | sed 's/, $//')
 
+if chroot binary dpkg -l | grep -q delphix-virtualization; then
```

Comment:
I think we need to set pipefail for this to handle the case where the dpkg command fails?

Reply:
We're already doing that, right? See [here](https://github.com/delphix/appliance-build/blob/master/live-build/misc/live-build-hooks/82-upgrade-repository.binary#L27); or am I overlooking something?

Reply:
Ah you are right, I missed that